views/components/Sidebar/SideNavigation.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { render, getByTestId } from '@testing-library/react'
+import SideNavigation from './SideNavigation'
+import { MemoryRouter } from 'react-router-dom'
+import { createUser } from '../../../test/fixtures'
+
+jest.mock('./SidebarLink')
+
+describe('SideNavigation', () => {
+  const user = createUser({ role: 'admin' })
+
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = render(
+      <MemoryRouter>
+        <SideNavigation user={user} />
+      </MemoryRouter>
+    )
+  })
+
+  test('renders SideNavigation component', () => {
+    const { container } = wrapper
+    const nav = getByTestId(container, 'nav')
+
+    expect(nav).toBeInTheDocument()
```

Comment:
we shouldn't need to set the `wrapper` element if we import `screen` from `@testing-library/react`, which is the recommended way to query the dom.

```suggestion
    const nav = screen.getByTestId('nav')

    expect(nav).toBeInTheDocument()
```

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)